<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sliderDialogClass</class>
 <widget class="QDialog" name="sliderDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sliderDialog</string>
  </property>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nx</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ux</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>σ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_scale">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>311</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
